config/eslint/base.cjs

```diff
@@ -20,13 +32,13 @@ function rules() {
     'prefer-rest-params': 'off',
     'prefer-const': 'error',
 
-    "prettier/prettier": [
-      "error",
+    'prettier/prettier': [
```

Comment:
NOTE: We currently don't lint for prettier (or anything AFAICT) in this part of the project. Should we?

Reply:
probably. I want to separate prettier out like we did in AB frontend, but haven't had a chance to. Would handle this.